packages/nr-metadata/nr_metadata/ui/semantic-ui/js/nr/forms/components/LicenseField/LicenseFieldItem.jsx

```diff
@@ -7,30 +7,63 @@
 // under the terms of the MIT License; see LICENSE file for more details.
 
 import React from "react";
-import { Button, Grid } from "semantic-ui-react";
+import { Button, Grid, Image } from "semantic-ui-react";
 import { i18next } from "@translations/nr/i18next";
 import PropTypes from "prop-types";
 import { useFormikContext } from "formik";
-import { getTitleFromMultilingualObject } from "@js/oarepo_ui";
+import { useFormConfig } from "@js/oarepo_ui";
 
 export const LicenseFieldItem = ({ license, fieldPath }) => {
   const { setFieldValue } = useFormikContext();
+  const {
+    formConfig: {
+      vocabularies: {
+        rights: { all },
+      },
+    },
+  } = useFormConfig();
+  const licenseUI = all.find((r) => r.value === license.id);
```

Comment:
This would not be needed if search app of LicenseSelect uses UI serializer for search results